test/BindTest.php

```diff
@@ -318,6 +318,11 @@ public function testSaslBindNoExplicitUsername()
         ];
         $ldap = $this->getSslLdap($options);
         $ldap->bind();
+        // The "pass" expectation here is just that no exception was thrown.
+        // Getting to this point in the code is the "definition of pass".
+        // phpunit will flag this as a risky test if we do not assert anything,
+        // so assert something.
+        $this->assertIsObject($ldap);
```

Comment:
Note: this kind of assertion will trip static analysis (to be introduced in future).

Is there anything else we can assert upon? For example getting something from `$ldap` and verifying its value, rather than its shape?

Reply:
I changed it to check the value of `getLastErrorCode()`